src/modules/simulation/simulator_sih/sih_params.c

```diff
@@ -235,33 +235,31 @@ PARAM_DEFINE_FLOAT(SIH_KDW, 0.025f);
  * Initial geodetic latitude
  *
  * This value represents the North-South location on Earth where the simulation begins.
- * A value of 45 deg should be written 450000000.
  *
  * LAT0, LON0, H0, MU_X, MU_Y, and MU_Z should ideally be consistent among each others
  * to represent a physical ground location on Earth.
  *
  * @unit deg*1e7
```

Comment:
This should be deg now

Reply:
right, thanks! changed it in latest commit